src/Tasks/Common/ConflictResolution/ConflictResolver.cs

```diff
@@ -349,6 +349,25 @@ private TConflictItem ResolveConflict(TConflictItem item1, TConflictItem item2,
                 return item2;
             }
 
+            if (item1.ItemType == ConflictItemType.CopyLocal && item2.ItemType == ConflictItemType.CopyLocal)
```

Comment:
I'm wondering if it's possible to have 3 items conflict, where the first one is not CopyLocal, and the second two ones are CopyLocal.  Then the CopyLocal items would not be compared against each other, and you could still have double writes.

Reply:
Do you know how I might repro that?

Reply:
No, I'm not sure if it's actually possible.  If it looks like it won't happen based on code inspection then we can leave it be.

Reply:
It appears to be possible if you have a `Reference` to this file and two copy local items. Trying to repro.

Reply:
I think it would have to be a Reference without Private=false, which would never come from a package.  Even so, not managing to repro.

Reply:
It still ends up comparing the two CopyLocal items in this case, still trying to understand if that's guaranteed.

Reply:
It can't happen for the publish case because we inky pass _ResolvedCopyLocalPublishAssets to ResolvePackageFileConflicts and they will all be CopyLocal and we will pick a winner among them. So it's definitely sufficient for the bug this claims to fix for publish. If you have a file reference to the exact same file version without Private="false", then there will be duplicates in ReferenceCopyLocalPaths when RAR finishes, though. I think this is edge enough to leave for another fight. I can't construct a real scenario.